### ISSUE TITLE
CUDA_ERROR_INVALID_VALUE

### ISSUE THREAD

**Issue by Cram3r95** [Maghoumi/pytorch-softdtw-cuda#26]
Hi guys,

Im trying to run SoftDTW between two tensors with the following dimensions:

bs x num_points x data_dimensionality

relevant_centerlines = 1024 x 30 x 2
predictions = 1024 x 30 x 2

As following:

error = sdtw(relevant_centerlines, predictions)

However, Im getting the following error. What can I do?

```
[INFO:  246]: init
[ERROR:  385]: Call to cuPointerGetAttribute results in CUDA_ERROR_INVALID_VALUE
*** numba.cuda.cudadrv.driver.CudaAPIError: [1] Call to cuPointerGetAttribute results in CUDA_ERROR_INVALID_VALUE
```

**Comment by yhchen1123** [comment]
Hi Carlos,

Im getting the same error. I wonder have you fixed it? And how?

Thanks a lot.

**Comment by Cram3r95** [comment]
Yes, I was @yhchen1123. The solutions are the versions. You must have a compatible CUDA version (in my case 11.6) and the associated dependencies. I am developing this based on this Docker image:

docker pull nvcr.io/nvidia/pytorch:22.04-py3

**Comment by yhchen1123** [comment]
Many thanks for the comment!
